report_async/static/src/xml/report_async.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name='ReportAsyncConfiguration'>
+        <div class="form">
+            <div class="form-group">
+                <label for="async-user-email">Email Address</label>
+                <input type="email" class="form-control" id="async-user-email"
+                       aria-describedby="emailHelp"
+                       placeholder="admin@example.com"/>
```

Comment:
Could you fill this in by default as `self.env.user.email` (I mean not the placeholder, that is fine like it is, but the actual value)? That will be the one they want in 99% of cases.